.github/workflows/connector-ci-checks.yml

```diff
@@ -170,6 +170,33 @@ jobs:
         working-directory: airbyte-integrations/connectors/${{ matrix.connector }}
         run: poe test-integration-tests
 
+      - name: Install Tailscale
+        if: failure()
+        shell: bash
+        run: |
+          curl -fsSL https://tailscale.com/install.sh | sh
+
+      - name: Join Tailnet
+        if: failure()
+        shell: bash
+        run: |
+          sudo tailscale up \
+            --authkey=${{ secrets.TAILSCALE_AUTHKEY }} \
+            --hostname=ci-runner-${{ github.run_id }} \
+            --ssh \
+            --advertise-tags=tag:ci \
+            --accept-routes=false \
+            --reset
+
+      - name: Sleep for SSH Debugging (Failure Only)
+        if: failure()
+        shell: bash
+        run: |
+          echo "Build failed. SSH access enabled via Tailscale."
+          echo "ssh command: ci-runner-${{ github.run_id }}"
+          echo "Sleeping for 3 hours for debugging access..."
+          sleep 7200
```

Comment:
@davinchia - What are your thoughts re: cost? Essentially, all failures will keep the compute running for three hours. If a developer is pushing commits regularly, and if tests aren't yet passing, presumably this would be close to a full day of compute on a large instance.

One alternative, if we're okay with this being an "on demand" debug option is that it could be the _default_ behavior when run via slash command (`/run-connector-tests`) but otherwise skipped.

I don't feel strongly - and this is Java-specific as of now - so I won't object if you still want this as "always on" behavior for all java connectors.